python-stdlib/enum/manifest.py

```diff
@@ -0,0 +1,3 @@
+metadata(version="0.0.1")

```

Comment:
I would suggest version 1.0.0 if this is stable to [allign with semver](https://victorpierre.dev/blog/beginners-guide-semantic-versioning/#when-and-why-to-use-semantic-versioning)
or at least 0.8.0 if you think is still needs testing and expect some bugs.

Reply:
Thanks